src/format/opamFile.ml

```diff
@@ -1781,15 +1854,18 @@ module Switch_configSyntax = struct
 
   let to_list = Syntax.to_list pp
 end
+
 module Switch_config = struct
   include Switch_configSyntax
   include SyntaxFile(Switch_configSyntax)
+  module BestEffort = MakeBestEffort(Switch_configSyntax)
 end
 
 module SwitchSelectionsSyntax = struct
 
   let internal = "switch-state"
   let format_version = OpamVersion.of_string "2.0"
+  let file_format_version = OpamVersion.of_string "2.0"
```

Comment:
Shouldn't this then be used on L1888 below?

Reply:
well spotted!